provisioner/plain-v0/README.md

```diff
@@ -137,17 +141,18 @@ Create the bundle:
 apiVersion: core.rukpak.io/v1alpha1
 kind: Bundle
 metadata:
-  name: combo-0.0.1
+  name: combo-v0.0.1
 spec:
-  image: quay.io/tyslaton/combo-plain-bundle:v0.0.1
+  image: quay.io/tflannag/bundle:combo-operator-v0.0.1
   provisionerClassName: core.rukpak.io/plain-v0
 ```
 
 Check the Bundle status via `kubectl get bundle combo-0.0.1`. Eventually the Bundle should show up as Unpacked.
 
-```
-NAME          IMAGE                                        PHASE      AGE
-combo-0.0.1   quay.io/tyslaton/combo-plain-bundle:v0.0.1   Unpacked   17h
+```bash
+$ kubectl get bundle combo-v0.0.1
+NAME           IMAGE                                           PHASE      AGE
+combo-v0.0.1   quay.io/tflannag/bundle:combo-operator-v0.0.1   Unpacked   38m
```

Comment:
we still have that issue with a stale Age coming through, I forgot to change it here as well 😄 Could we change it to be like 10s in all the examples?

Reply:
Yeah that sounds reasonable to me - updated. We can always change this in the future as a follow up as well.